### ISSUE TITLE
Combobox becomes unresponsive if searching for a non-existing entity

### ISSUE THREAD

**Issue by Justin991q** [home-assistant/frontend#13336]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

If you search for an non-existing icon or entity in a combo box and then correct it with an existing entity then the combo box does not work anymore.
The combo box shows up but you are unable to click or scroll through the list.

### Describe the behavior you expected

If you search for a non-existing entity and remove the text and type the correct entity you should be able to click the entity.

### Steps to reproduce the issue

```
1. Go to the dashboard editor and add/edit a card
2. Type in something random in the entity field 
3. Remove the text (ctrl + a) and type in an entity that exists
4. You see the combo box but won't be able to use it.
```

https://user-images.githubusercontent.com/4533867/182821895-1e3cc303-c6d2-4042-9e50-845fa283671e.mp4

### What version of Home Assistant Core has the issue?

Home Assistant 2022.8.0, Frontend 20220802.0

**Comment by steverep** [comment]
Are you able to test the current development branch? It includes a dependency upgrade for combo boxes that fixed several bucks.

**Comment by Justin991q** [comment]
Hi,

I just tried the latest development branch and the issue still exists. I also tried the dev branch on a clean install to double check but the issue is also on a clean install.

**Comment by steverep** [comment]
Thanks for following up. Can you please state what browser and OS combinations you've tested?

**Comment by Justin991q** [comment]
I've tried it on Firefox, Brave and Chrome on Windows 11 and it seems to be working fine on Firefox.
Also on the Android companion app the combobox breaks if you type in an non-existing entity

**Comment by piitaya** [comment]
We pushed some fixes with the `2022.10` release. Is the problem still there?

**Comment by Justin991q** [comment]
It looks like this issue has been solved, thanks!
